src/components/Users/UsersList.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { useUsersContext } from '../../hooks/useUsersContext';
+import { User } from '../../types';
+
+type Props = {
+  onCloseDropdown: () => void;
+};
+
+export const UsersList: FC<Props> = ({ onCloseDropdown }) => {
+  const { onSelectUser, selectedUser, users } = useUsersContext();
```

Comment:
Check if the functions and variables from useUsersContext are properly defined and exported. If not, it might cause a runtime error.